DCO

```diff
@@ -0,0 +1,37 @@
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
```

Comment:
I'm assuming this means that LF developed the DCO and its their thing, hence the copyright; is that correct?

Reply:
Yes, this is how I understand it.